amx/server/syscalls.lua

```diff
@@ -2788,7 +2810,7 @@ function HTTP(amx, index, type, url, data, callback)
 			procCallInternal(amx, callback, index, error, responseData)
 			return 0;
 		end
-	end, data, false)
+	end)
 	return 0;

```

Comment:
`return 0` is always returned. The `return 0` and `return 1` in the callback (in the lines above, next to `procCallInternal`) never actually get returned.

It should be something like:

```
local success = fetchRemote(...)

if not success then
    return 0
end
return 1
```

And the `return 1;` and `return 0;` inside the callback aren't necessary (the callback return value is unused by MTA!)